### PR TITLE
Adds initial guidelines for security reviewers

### PR DESCRIPTION
**Pull request by gunsch**
First pass at a list of "here are questions you should ask in privacy/security/infrastructure review".

PTAL, let me know if I'm missing anything obvious.

### CODE REVIEW

File: platform/security/hosting-a-security-review.md
Comment (rianfowler) [inline platform/security/hosting-a-security-review.md]:
```suggestion
* What kinds of information are logged in the new logging? Is there anything that uniquely identifies a user (Personally Identifiable Information)?
```

File: platform/security/hosting-a-security-review.md
Comment (rianfowler) [inline platform/security/hosting-a-security-review.md]:
```suggestion
* Are there any new logging statements? (FE or BE). This includes data sent to Sentry, Cloudwatch, Google Analytics, ...?
```

File: platform/security/hosting-a-security-review.md
Comment (rianfowler) [inline platform/security/hosting-a-security-review.md]: We need to train FE on these

**Review by rianfowler — APPROVED** [review]

**Comment by gunsch** [comment]
@johnpaulashenfelter @wyattwalter any thoughts? I'll plan to merge this tomorrow if I don't hear any other feedback.

**Comment by gunsch** [comment]
Added some questions about rollout plan (from this morning's security review). Merging!

**Pull request merged**
